native/src/components/DatePickerInput.tsx

```diff
@@ -1,11 +1,12 @@
 import { DateTime } from 'luxon'
 import React, { ReactElement, RefObject } from 'react'
 import { NativeSyntheticEvent, StyleProp, TextInput, TextInputKeyPressEventData, ViewStyle } from 'react-native'
+import { KeyboardExtendedInput } from 'react-native-external-keyboard'
 import styled, { useTheme } from 'styled-components/native'
 
 import { zeroPad } from 'shared/utils/dateFilterUtils'
 
-const Input = styled(TextInput)`
+const Input = styled(KeyboardExtendedInput)`
```

Comment:
💭 Looks like this won't be needed with RN 80 (https://github.com/ArturKalach/react-native-external-keyboard?tab=readme-ov-file#react-native-external-keyboard), so maybe we add a comment for that and mention it in the next RN update issue (do we already have that?)